src/models/Attachment.ts

```diff
@@ -0,0 +1,6 @@
+export default interface Attachment {
+    URL: string,
+    pos: number,
+    name: string,
```

Comment:
I am sorry, but name seems to be more for humans, and it feels weird for an attachment

Reply:
I left this attribute name because it is the one from the Trello-route file.